packages/core/src/Services/Metadata/MetadataResolver.ts

```diff
@@ -27,6 +27,13 @@ export class MetadataResolver {
     return url;
   }
 
+  public resolveNamespace(params: MetadataTypes.ResolveNamespaceParams): string {
```

Comment:
`MetaResolver` is already exported from `core`, so you can resolve it directly inside the WebSocket package.
You could extend the `__metadata` structure by adding a new field like `[key: string]: Record<string, unknown>`, and then set your custom values inside the WebSocket service.

We shouldn’t add extra logic to `core`, especially for performance reasons - users who don’t use `wss` would end up getting additional checks that aren’t relevant for them.